### PR TITLE
Create service for Pivotal Tracker api

### PR DESCRIPTION
**Pull request by cleicar**
This PR solves #65

### CODE REVIEW

**Comment by cleicar** [comment]
@bronzdoc @FionaDL do you guys have time to review this PR? Thanks!

File: app/workers/pivotal_worker.rb [hunk 1]
```diff
@@ -1,47 +1,15 @@
...
 
   def perform(user_id)
     user = User.find(user_id)
 
-    @client = TrackerApi::Client.new(token: user.pivotal_token)
+    pivotal_service = PivotalApiService.new(user.pivotal_token)
 
     @first_run = user.pivotal_id.nil?
-    user.update(pivotal_id: @client.me.id) unless user.pivotal_id
+    user.update(pivotal_id: pivotal_service.user_pivotal_id) if @first_run
```
Comment (bronzdoc) [inline app/workers/pivotal_worker.rb:11]: Don't we want to have the same logic here as before? with the new logic, we will run the update if `user.pivotal_id` is `nil` ?

**Comment by cleicar** [comment]
@bronzdoc this is ready to review again :)

**Review by bronzdoc — APPROVED** [review]

**Comment by bronzdoc** [comment]
@ABizzinotto this is ready for QA

**Comment by ABizzinotto** [comment]
@bronzdoc @cleicar looks good

**Comment by bronzdoc** [comment]
@cleicar this just needs a rebase and we good to go!
https://minnesotareformer.com/wp-content/uploads/2020/03/little-kid-dancing-.gif

**Comment by cleicar** [comment]
This PR can be closed, because we already did a refactor in the PivotalWorked on PR https://github.com/fastruby/dash/pull/46

**Pull request closed without merging**